src/CommandingOfficers/SturmValueFinders.java

```diff
@@ -0,0 +1,46 @@
+package CommandingOfficers;
+
+import Units.Unit;
+import Units.UnitModel;
+
+public abstract class SturmValueFinders // Until such time as we find out AW1 works substantially differently
+{
+  public static class HPValueFinder implements IValueFinder
+  {
+    public boolean countHidden = true; // Ignore cloaked units even if we can see them
```

Comment:
Value says "yes, we will count hidden units", but comment seems to indicate "no, we shouldn't count them."

They say that if the code and comments disagree, they are both wrong. Since this is a Boolean expression, that's literally impossible here, but y'know, it's the principle of the thing.

Reply:
Comment is missing a few words of context. Fair enough.